### ISSUE TITLE
Consider using Infra

### ISSUE THREAD

**Issue by annevk** [w3c/webdriver#1529]
In particular for JSON there are some nice primitives at https://infra.spec.whatwg.org/#json that we already use elsewhere in the browser that seem good to align on.

**Comment by foolip** [comment]
Yep, I've had the same thought. @domenic has also pointed me to https://github.com/whatwg/infra/issues/159, some of which we might make use of.
